<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
    <item>
     <widget class="QLabel" name="lblAppStatus">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QPushButton" name="cmdDiscover">
         <property name="text">
          <string>Discover</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkAutoDetect">
         <property name="text">
          <string>Auto open</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cmdAIn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Analog In</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdAOut">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Analog Out</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdDIn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Digital In</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdDOut">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Digital Out</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdCtr">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Counters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdMisc">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Misc</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdConfig">
         <property name="text">
          <string>Info/Conf</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdHistory">
         <property name="text">
          <string>History</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>905</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionNOSCALEDATA"/>
    <addaction name="actionNOCALIBRATEDATA"/>
    <addaction name="actionEXTCLOCK"/>
    <addaction name="actionEXTTRIGGER"/>
    <addaction name="actionCONTINUOUS"/>
    <addaction name="actionBACKGROUND"/>
    <addaction name="separator"/>
    <addaction name="actionTmrloop"/>
   </widget>
   <widget class="QMenu" name="menuTriggering">
    <property name="title">
     <string>Triggering</string>
    </property>
    <addaction name="actionRISING_EDGE"/>
    <addaction name="actionFALLING_EDGE"/>
    <addaction name="actionACTIVE_HIGH"/>
    <addaction name="actionACTIVE_LOW"/>
    <addaction name="separator"/>
    <addaction name="action_SOURCE_LOCAL"/>
    <addaction name="actionSOURCE_MASTER"/>
    <addaction name="actionSOURCE_SLAVE"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionVolts_vs_Time"/>
   </widget>
   <widget class="QMenu" name="menuFunction">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Function</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBoards">
    <property name="title">
     <string>Boards</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRange">
    <property name="title">
     <string>Range</string>
    </property>
    <addaction name="actionBIP10V"/>
    <addaction name="actionBIP5V"/>
    <addaction name="actionBIP2V"/>
    <addaction name="actionBIP1V"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Queue"/>
   </widget>
   <addaction name="menuBoards"/>
   <addaction name="menuFunction"/>
   <addaction name="menuOptions"/>
   <addaction name="menuRange"/>
   <addaction name="menuTriggering"/>
   <addaction name="menuPlot"/>
  </widget>
  <action name="actionNOSCALEDATA">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NOSCALEDATA</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionNOCALIBRATEDATA">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NOCALIBRATEDATA</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionEXTCLOCK">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>EXTCLOCK</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionEXTTRIGGER">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>EXTTRIGGER</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionCONTINUOUS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CONTINUOUS</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionBURSTMODE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BURSTMODE</string>
   </property>
  </action>
  <action name="actionBACKGROUND">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BACKGROUND</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionRISING_EDGE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RISING_EDGE</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionFALLING_EDGE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FALLING_EDGE</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionACTIVE_HIGH">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ACTIVE_HIGH</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionACTIVE_LOW">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ACTIVE_LOW</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionVolts_vs_Time">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Volts vs Time</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionLoad_Queue">
   <property name="text">
    <string>Load Queue</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionTmrloop">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Timer loop</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_SOURCE_LOCAL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SOURCE_LOCAL</string>
   </property>
  </action>
  <action name="actionSOURCE_MASTER">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SOURCE_MASTER</string>
   </property>
  </action>
  <action name="actionSOURCE_SLAVE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SOURCE_SLAVE</string>
   </property>
  </action>
  <action name="actionBIP10V">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BIP10V</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionBIP5V">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BIP5V</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionBIP2V">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BIP2V</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionBIP1V">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BIP1V</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
